README.md

```diff
@@ -64,6 +66,18 @@ Basic configuration options are detailled [here](https://kubernetes.github.io/in
 
 You can find your external MongoDB prerequisites [here](/doc/mongo-prerequisites.md).
 
+#### OpenShift
+
+You have a specific values-custom to ease the deployment of NeoLoad Web on OpenShift: values-custom-openshift.yaml.
+
+In this file you will have to modify the security context parameters to match your cluster rights.
+You can find documentation on security context [here](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/).
+
+To enable TLS on your routes you should:
+- Have a valid certificate.
+- Fill your certificate in the values-custom-openshift file 
+- Or reference the name of the secret that holds that certificat (by default in OpenShift there is a router-certs secret in the default workspace).
```

Comment:
We should make this sentence less ambiguous.
As it is it can be interpreted as 'you can use the router-certs as value.
But we know we cannot reference secrets from another namespace.

Also workspace is not the right word here: it should be namespace (the k8s terminology) or project (the openshift terminology).